assets/css/style.scss

```diff
@@ -2,4 +2,6 @@
 # Only the main Sass file needs front matter (the dashes are enough)
 ---
 
-@import "minima-{{ site.minima.skin | default: 'classic' }}";
+@import
+  "minima-{{ site.minima.skin | default: 'classic' }}",
```

Comment:
Since `minima-classic.scss`, `minima-solarized.scss`, and `minima-solarized-dark.scss` are only color-related, I think put these files into `_sass/minima/skins/` would make their purpose clear. The file structure looks like below:

```
_sass/
  minima/
    skins/
      classic.scss
      solarized.scss
      solarized-dark.scss
    _base.scss
    _layout.scss
    initialize.scss
```

Then this line is modified to 

```scss
@import
  "minima/skins/{{ site.minima.skin | default: 'classic' }}",
```

Reply:
This is a good suggestion. Therefore, I'll let you take full credit for it.
Feel free to submit a PR with the change and associated updates to the README after this PR gets merged.

Reply:
OK